adapters/beachfront/beachfronttest/supplemental/adm-video-app-malformed-bundle.json

```diff
@@ -0,0 +1,123 @@
+{
+  "mockBidRequest": {
+    "id": "adm-video",
+    "imp": [
+      {
+        "id": "video1",
+        "ext": {
+          "bidder": {
+            "bidfloor": 3.01,
+            "appId": "videoAppId1"
+          }
+        },
+        "video": {
+          "mimes": [
+            "video/mp4"
+          ],
+          "context": "instream",
+          "w": 300,
+          "h": 250
+        }
+      }
+    ],
+    "app": {
+      "bundle": "some.domain.us/some/page.html"
+    },
+    "device":{
+      "ip":"192.168.168.168"
+    }
+  },
+
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "https://qa.beachrtb.com/bid.json?exchange_id=videoAppId1",
+        "body": {
+          "id": "adm-video",
+          "imp": [
+            {
+              "video": {
+                "w": 300,
+                "h": 250,
+                "mimes": [
+                  "video/mp4"
+                ]
+              },
+              "bidfloor": 3.01,
+              "id": "video1",
+              "secure": 0
+            }
+          ],
+          "app": {
+            "bundle": "qwerty"
```

Comment:
@muncha thanks for the clarification. I pushed your commits on this branch but I believe this test will fail because based on the malformed `bundle` above, the domain here will get set to: `domain.some`. It might be worth it to add some validation for the app bundle in the code in order to avoid this

Reply:
@mansinahar 
Please try again with the most recent commit, which is the one that I thought was up there, but apparently it was not.

Reply:
@mansinahar 
"It might be worth it to add some validation for the app bundle in the code in order to avoid this"
Agreed.

Just Googled a bit, and the Android bundle is not required to start with a TLD, which might be an expensive thing to validate completely without false positives. Say someone has an app developed by the Vatican - [.va](https://en.wikipedia.org/wiki/.va) has to pass, but what about [.lu](https://en.wikipedia.org/wiki/Proposed_top-level_domain#Geographic_proposals)? 

It's much [simpler](https://developer.android.com/studio/build/application-id#:~:text=It%20must%20have%20at%20least,zA%2DZ0%2D9_%5D.) than that.

Maybe this suggests some validation at an application level?

Reply:
@muncha Well, Prebid Server just passes the app bundle information as is. In this case the beachfront adapter is using that to resolve the app domain and if sending an invalid app domain would cause the request to be marked invalid on your end then I'd say it'd be the beachfront adapter's responsibility to do that validation. If this won't affect anything on your side and you're okay with an invalid domain being sent in case of an invalid app bundle then the way you have the code right now should be fine.

Reply:
Cool. I'll check what effect an invalid domain may have on our ad server end. I think it is not a problem, but if it is I'll put an issue in and get on it.

Reply:
@mansinahar ...or just add it in here if needed, if you think that's better.

Reply:
@mansinahar 
I went ahead and did a bundle validation branch off of this one:
https://github.com/beachfront/prebid-server/tree/validateBundle

It seems to do what it needs to do. Maybe you want to merge it in here, maybe I do a PR once this is merged?

I still need to find out from beachfront if I should delete seemingly invalid bundle values or just pass it on, plus a test or two.

Reply:
@mansinahar 
They say if I can't figure out what it is, pass it anyway. I'll be updating the validateBundle branch shortly.

Reply:
@muncha So let's get this one in and you can do a follow-up PR for validation as needed. We are cutting out a tag today and I would like to get this in if possible